.gitignore

```diff
@@ -171,9 +171,6 @@ node_modules/
 # Yarn Integrity file
 .yarn-integrity
 
-# dotenv environment variables file
-.env
-
```

Comment:
Curious why you're removing `.env` from the gitignore? I would recommend keeping it and adding a `.env.example` like on the dashboard